jupyterhub/app.py

```diff
@@ -1670,9 +1670,12 @@ def init_hub(self):
         # This lets whitelist be used to set up initial list,
         # but changes to the whitelist can occur in the database,
         # and persist across sessions.
+        total_users = 0
         for user in db.query(orm.User):
             try:
-                await maybe_future(self.authenticator.add_user(user))
+                f = self.authenticator.add_user(user)
+                if f:
+                    await maybe_future()
```

Comment:
for my education: why doesn't the `f` go in the `maybe_future`? Right now it seems like it is a future that isn't awaited, which is a bug??

Or is it because we don't need to/care to wait for the future to resolve. Feels like there should be a way to make that explicit (or a comment?). The `await maybe_future()` is then just a way of saying "I yield, in case someone else wants to do something"?

Reply:
>  why doesn't the f go in the maybe_future

Because I forgot to put it there. It definitely needs to be there!